pkg/cloud/rgraph/rnode/backendservice/node.go

```
@@ -81,12 +86,38 @@ func (n *backendServiceNode) Diff(gotNode rnode.Node) (*rnode.PlanDetails, error
 		}, nil
 	}
 	return &rnode.PlanDetails{
-		Operation: rnode.OpRecreate,
+		Operation: rnode.OpUpdate,
 		Why:       "BackendService needs to be updated: " + strings.Join(details, ", "),
 		Diff:      diff,
 	}, nil
 }
 
+func fingerprint(gotNode *backendServiceNode) (string, error) {
+	gotRes := gotNode.resource
+	switch gotRes.Version() {
+	case meta.VersionGA:
+		obj, err := gotRes.ToGA()
+		if err != nil {
+			return "", fmt.Errorf("Update function failed, cannot fetch fingerprint from GA object, err: %v", err)
```

I would just propagate the error unchanged for this (same comment below)

`return "", err`

---

done